Source/WebKit/WebProcess/WebPage/RemoteLayerTree/PlatformCALayerRemoteTiledBacking.h

```diff
@@ -38,7 +38,7 @@ class PlatformCALayerRemoteTiledBacking final : public PlatformCALayerRemote {
 private:
     PlatformCALayerRemoteTiledBacking(WebCore::PlatformCALayer::LayerType, WebCore::PlatformCALayerClient* owner, RemoteLayerTreeContext&);
 
-    WebCore::TiledBacking* tiledBacking() override { return m_tileController.get(); }
+    WebCore::TiledBacking* tiledBacking() final { return m_tileController.ptr(); }
```

Comment:
Is this something that is now never null for all classes derived from PlatformCALayer?

Reply:
Thanks for the review!

I don't really know how all of this fits together unfortunately. Maybe @smfr or @hortont424 would know follow-up cleanup that's possible.

I did look around a bit and there's not that many instances of `tiledBacking()` overall, so maybe something can be done.